Source/ThirdParty/enet/enet.h

```diff
@@ -5002,7 +4991,7 @@ extern "C" {
             tv->tv_nsec = t.QuadPart % 1000000 * 1000;
             return (0);
         }
-    #elif __APPLE__ && defined(__MAC_OS_X_VERSION_MIN_REQUIRED) && __MAC_OS_X_VERSION_MIN_REQUIRED < 101200
```

Comment:
This `defined(__MAC_OS_X_VERSION_MIN_REQUIRED)` check was required when cross-compiling from macOS to iOS